Tests/MongoSwiftTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -248,11 +266,18 @@ struct UnifiedTestRunner {
                             }
                         }
                     }
+                    // Manual clean up
+                    try await self.terminateOpenTransactions()
+                    await context.disableFailpoints()
+                    try await self.closeEntities(context: context)
                 } catch let testErr {
                     // Test runners SHOULD terminate all open transactions after each failed test by killing all
                     // sessions in the cluster.
                     do {
-                        try self.terminateOpenTransactions()
+                        // Clean up to prevent a client closing error from obscuring the real error
+                        try? await self.terminateOpenTransactions()
+                        await context.disableFailpoints()
+                        try await self.closeEntities(context: context)
                     } catch {
                         print("Failed to terminate open transactions: \(error)")
```

Comment:
This print statement seems outdated now, since we run `terminateOpenTransactions` with `try?`.

Reply:
updated to be `print("Failed to close unified test runner elements: \(error)")`